### ISSUE TITLE
Setting a video key on Video Recorder Component

### ISSUE THREAD

**Issue by annaPerdomo** [Ziggeo/react-ziggeo#68]
Hello all! 

I ran into an issue with react-ziggeo within the React/Next.Js/Redux application that I'm working on and I wanted to check and make sure that I'm doing everything correctly on my side. Here is some more information regarding the versions of some of the packages I'm working with: 
react-ziggeo: 4.3.0
react: 16.9
next.js: 9.4.4

Since we're using Next.JS to render our application on both the client and server side, the Video Recorder is being imported into the react functional component like this: 

```
import dynamic from 'next/dynamic';
const VideoRecorder = dynamic(
  () => import('react-ziggeo').then((app) => (app.ZiggeoRecorder)),
  { ssr: false}
);
```
What I ultimately want to be able to do is to define and set a key for each video before the video has finished processing. I can already define and set a video title before processing with the code below that is in the render function of the React functional component:
```
<VideoRecorder
  title={'test_title'}
/>
```
I would like to set the video key in the same way but it doesn't work and the video's key is still null after the video has been processed: 
```
<VideoRecorder
  key={'test_key'}
/>
```
Since the key property is listed on the [Embedding Parameters documentation](https://ziggeo.com/docs/sdks/javascript/browser-integration/parameters#javascript-revision=stable) under Media Management parameters, I thought it might be possible. 

I know that I can use the server SDK to add a key after the fact but I would like to see if it's possible to do it before processing. 

Thank you for your help!

**Comment by sambua** [comment]
Hello @annaPerdomo ,

As `key` keyword is reserved in React, we will upgrade soon with new version where you can use `_key` instead. 

Will update this issue as soon as changes will be applied.

**Comment by annaPerdomo** [comment]
Hello @sambua!

Sounds great! I appreciate your fast response and look forward to seeing the change in future versions!

**Comment by sambua** [comment]
Hello @annaPerdomo ,

Could you please try to use latest version `4.3.1` and advice? You can use `_key` property instead of Ziggeo `key` property.

**Comment by sambua** [comment]
Should be fixed with #69

**Comment by annaPerdomo** [comment]
Hello @sambu! 

Today I just got back in the office so I apologize for my late response.  I have tried out the new version and the `_key` property works beautifully! If I happen to run into another issue I'll go ahead and submit a new one, but so far everything works great. 

Thank you for addressing this issue so quickly!
